src/components/Navbar.jsx

```diff
@@ -11,6 +13,23 @@ class Navbar extends Component {
 
   render() {
     const notMobile = { minWidth: Responsive.onlyMobile.maxWidth + 1 }
+    let logOut, logIn
+
+    if (this.props.currentUser.isSignedIn) {
```

Comment:
I would turn this into an `if else` or a ternary operator

Reply:
We agree! :)